packages/plugin-ext/src/hosted/browser/worker/worker-main.ts

```diff
@@ -85,7 +85,7 @@ const pluginManager = new PluginManagerExtImpl({
     async init(rawPluginData: PluginMetadata[]): Promise<[Plugin[], Plugin[]]> {
         const result: Plugin[] = [];
         const foreign: Plugin[] = [];
-        for (const plg of rawPluginData) {
+        await Promise.all(rawPluginData.map(async plg => {
```

Comment:
@tsmaeder was this loop supposed to happen sequentially, or is it ok to parallelize?

Reply:
I did not think about that. The only possible negative consequence I can think of is that the order of plugins in the result may be changed.

Reply:
Is the order of the plugins in the result important?

Reply:
I have no idea.

Reply:
The order is now conserved.